jenkins/bootstrap.py

```diff
@@ -333,29 +334,42 @@ def node():
 
 
 def find_version():
-    """Determine and return the version of the build."""
-    version_file = 'version'
```

Comment:
it's written when the tarball is created - https://github.com/kubernetes/kubernetes/blob/master/build-tools/lib/release.sh#L428.

it should be the same as the `GitVersion` in `kubectl version` - but since we're parsing that already, it didn't seem like reading another file was necessary anymore.